test/api/v3/unit/libs/cron.test.js

```diff
@@ -313,6 +313,24 @@ describe('cron', () => {
       expect(tasksByType.dailys[0].completed).to.be.false;
       expect(user.stats.hp).to.equal(healthBefore);
     });
+
+    it('sets isDue for daily', () => {
+      let daily = {
+        text: 'test daily',
+        type: 'daily',
+        frequency: 'daily',
+        everyX: 5,
+        startDate: new Date(),
+      };
+
+      let task = new Tasks.daily(Tasks.Task.sanitize(daily)); // eslint-disable-line new-cap
+      tasksByType.dailys.push(task);
+      tasksByType.dailys[0].completed = true;
```

Comment:
In the manual tests I did, the bug did not apply to completed Dailies. Possibly the act of uncompleting it triggers the re-calculation of the due status? If you want me to, I'll confirm that with more manual tests locally using an old commit. If I'm not misremembering what I saw, then I think a test for each case (completed and not completed) might be safest.